apps/web/src/services/cost-explorer/CostExplorerLNB.vue

```diff
@@ -103,7 +104,15 @@ const state = reactive({
     showMoreQuerySetStatus: true,
     showFavoriteOnly: false,
 });
-
+const dataSourceState = reactive({
+    plugins: computed<PluginReferenceMap>(() => store.getters['reference/pluginItems']),
+    items: computed<MenuItem[]>(() => costQuerySetState.dataSourceList.map((dataSource) => ({
+        name: dataSource.data_source_id,
+        label: dataSource.name,
+        imageUrl: dataSourceState.plugins[dataSource.plugin_info?.plugin_id]?.icon,
```

Comment:
Question: How do you deal with cases without icons?

Reply:
Then.. just no-icon dataSource..
<img width="274" alt="image" src="https://github.com/cloudforet-io/console/assets/83635051/5cedf4ab-fa60-4e75-9fd0-13c15406a411">